common/util.go

```diff
@@ -440,19 +442,62 @@ func ReadAtMost(r io.Reader, n int) ([]byte, error) {
 	return b, err
 }
 
-func detectNvidiaDevices() ([]string, error) {
+func getGPU() ([]*gpu.GraphicsCard, error) {
 	gpu, err := ghw.GPU()
+
 	if err != nil {
 		return nil, err
 	}
 
+	return gpu.GraphicsCards, nil
+}
+
+func getPCI() ([]*pci.Device, error) {
+	pci, err := ghw.PCI()
+
+	if err != nil {
+		return nil, err
+	}
+
+	return pci.ListDevices(), nil
+}
+
+var GetGPU = getGPU
```

Comment:
Could you make these variables package-private if they are used only for tests? Meaning, make the first letter lowercase. Like this.

```go
var getGpu = getGpuDefault
var getCpu = getCpuDefault
```